test/jdk/javax/swing/JTable/KeyBoardNavigation.java

```diff
@@ -224,13 +173,127 @@ public void setValueAt(Object aValue, int row, int column) {
     }
 
     public static void main(String[] args) throws Exception {
-        SwingUtilities.invokeAndWait(() -> {
-            try {
-                initTest();
-            } catch (Exception e) {
-                throw new RuntimeException(e);
-            }
-        });
-        passFailJFrame.awaitAndCheck();
+        String INSTRUCTIONS = """
+                Instructions to Test:
+                1. Refer the below keyboard navigation specs
+                 (referenced from bug report 4112270).
+                2. Check all combinations of navigational keys in all four modes
+                  shift and control verifying each change to the selection against
+                  the spec. If it does, press "pass", otherwise press "fail".
```

Comment:
"...in all four modes shift and control verifying each change to the selection against the spec." 
Can this sentence be rephrased for clarity ?

Reply:
Updated.